### PR TITLE
get all events endpoint

### PR DESCRIPTION
**Pull request by KTong821**
## Purpose
Added the get all events endpoint. An event currently has only "name" and "description" properties.

## Approach
Copy-pasted clubs endpoint, replaced "club" with "event". 

## Testing
Added corresponding test cases from Clubs to Events.

### CODE REVIEW

File: server/src/migrations/Migration20210220195842.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Migration } from '@mikro-orm/migrations';
+
+export class Migration20210220195842 extends Migration {
+
+  async up(): Promise<void> {
+    this.addSql('create table "event" ("id" varchar(255) not null, "created_at" timestamptz(0) not null, "updated_at" timestamptz(0) not null, "name" varchar(255) not null, "description" varchar(255) not null);');
+    this.addSql('alter table "event" add constraint "event_pkey" primary key ("id");');
+
+    this.addSql('alter table "club" drop column "is_deleted";');
```
Comment (jeffzh4ng) [inline server/src/migrations/Migration20210220195842.ts:9]: Why are we altering the `club` table here?

**Review by jeffzh4ng — CHANGES_REQUESTED** [review]

File: server/src/migrations/Migration20210220195842.ts (hunk 1 above)
Comment (KTong821) [reply server/src/migrations/Migration20210220195842.ts:9]: This fixes the "is_deleted" column under clubs that was accidentally added from the get-all-clubs PR. Making a new PR to clean up the clubs migration mistake.

File: server/src/migrations/Migration20210220195842.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { Migration } from '@mikro-orm/migrations'
```
Comment (xujustinj) [inline server/src/migrations/Migration20210220195842.ts:1]: this file looks like it got formatted by Prettier

**Review by xujustinj — APPROVED** [review]
lgtm otherwise

File: server/src/migrations/Migration20210220195842.ts (hunk 2 above)
Comment (KTong821) [reply server/src/migrations/Migration20210220195842.ts:1]: fixed in latest commit.

**Review by xujustinj — APPROVED** [review]

**Review by jeffzh4ng — APPROVED** [review]

**Pull request merged**